### ISSUE TITLE
pandas.options.display.max_rows is set to "None" in __init__.py

### ISSUE THREAD

**Issue by tlml** [jgieseler/solarmach#18]
pd.options.display.max_rows is set to "None" in __init__.py:

https://github.com/jgieseler/solarmach/blob/4d81e25df8e237a2af8278441354ac17c2dcbea7/solarmach/__init__.py#L24-L29

This causes trouble for users who might want to use pandas and print bits of large dataframes (e.g. SEP data) after using solarmach in the same notebook: The whole dataframe is printed, swamping the whole notebook session...

Is the pd.options.display.max_rows=None really needed? The default seems to be 60, I'd be hard-pressed to find more than 60 spacecraft for solarmach to handle...

Likewise, the display format is set, which could cause annoyance (but not as big trouble as the max_rows=None).

There are also a couple of plt rcParams there that could be used only temporally, restoring back to whatever values were used outside of the solarmach calls.

**Comment by jgieseler** [comment]
Yes, you're right. I actually don't know why the `pd.options.display.max_rows` is set like this because as you said, there shouldn't be such big dataframes in solarmach. That change dates back more than 2 years, so it was before my time. But I remove it now, and will also save the plotting parameters in the beginning and reset them in the end.

**Comment by jgieseler** [comment]
Fixed with https://github.com/jgieseler/solarmach/commit/fbf3939f4ce59883ec8e5f60f021a39928551e6a & https://github.com/jgieseler/solarmach/commit/b29593baf2ed43b15542dcfc1801ee74b637473d

I also removed the permanent changes to `pd.options.display.float_format` and sunpy's log level with https://github.com/jgieseler/solarmach/commit/677792041840c9fa17ce093ecd63b6f5dadaa793

**Comment by jgieseler** [comment]
Fixed now in PyPI version: https://github.com/jgieseler/solarmach/releases/tag/v0.1.4 (conda-forge following later)
